java/statestore/src/main/java/sleeper/statestore/s3/S3FileReferenceStore.java

```diff
@@ -93,6 +94,8 @@ class S3FileReferenceStore implements FileReferenceStore {
     private final S3StateStoreDataFile<List<AllReferencesToAFile>> s3StateStoreFile;
     private Clock clock = Clock.systemUTC();
 
+    // Suppressing because false positives are found on method references/functional interfaces
+    @SuppressFBWarnings("CT_CONSTRUCTOR_THROW")
```

Comment:
This isn't a false positive, Objects.requireNonNull throws an exception.

Reply:
Spotbugs only seems to pick up on checked exceptions being thrown, so it ignores these calls.